src/app/toolbar/toolbar.component.html

```diff
@@ -15,21 +15,24 @@
 -->
 
 <mat-toolbar class="toolbar">
-  <mat-toolbar-row class="toolbar-row">
+  <mat-toolbar-row class="toolbar-row" [ngClass]="isMobileMenuOpen ? 'hidden-nav': ''  ">
     <a routerLink="/home" class="title-wrapper">
       <div class="icon-placeholder"></div>
       <span class="title">SafeNet</span>
     </a>
-    <ng-container *ngIf="!userIsSignedIn">
-      <span class="spacer"></span>
-      <button mat-button class="action-button get-started-button" routerLink="/">Get Started</button>
-    </ng-container>
+
+
+    <button mat-button class="action-button mobile-menu-button" (click)="toggleMobileMenu()">
+      <mat-icon svgIcon="hamburger_menu"></mat-icon>
+    </button>
+
```

Comment:
```suggestion
    <button mat-button class="action-button mobile-menu-button" (click)="toggleMobileMenu()">
      <mat-icon svgIcon="hamburger_menu"></mat-icon>
    </button>
```

Why so many empty lines?